subsys/bluetooth/controller/Kconfig

```
@@ -394,6 +394,17 @@ config BT_CTLR_SDC_EVENT_TRIGGER
 	  When used for connections, the connection event trigger can be configured to trigger
 	  every N connection events starting from a given connection event counter.
 
+config BT_CTLR_SDC_CONN_ANCHOR_POINT_REPORT
+	bool "Connection event anchor point reports"
+	depends on BT_CONN
+	help
+	  Enable support for obtaining connection anchor point reports.
+	  This is a proprietary SDC feature which allows the controller
+	  to produce timestamps for ACL connection anchor points.
+	  A new timestamp is generated whenever a connection's anchor point changes.
+	  The reporting is enabled through an HCI command.
+	  See Core_v5.4, Vol 6, Part B, Section 4.5.1.
```

```suggestion
	  See Bluetooth Core Specification, Vol 6, Part B, Section 4.5.1.
```
I think it's guaranteed now that section numbers will not change in the future spec versions.
This way we don't need to update it when new spec comes and avoid questions/complains from customers that we are not on the latest spec.

---

Are you ok with changing this in a separate PR to unblock the sample application PR? Then we can address all occurrences of similar patterns.